src/constants.tsx

```diff
@@ -96,7 +97,7 @@ export const STRUCTURE_LAYER = 'structure-layer';
 export type STRUCTURE_LAYER = typeof STRUCTURE_LAYER;
 export const PROGRESS = 'Progress';
 export type PROGRESS = typeof PROGRESS;
-export const STRUCTURES = 'structure(s)';
+export const STRUCTURES = `${STRUCTURES_UNIT_TYPE}(s)`;
```

Comment:
Will this switch everything to be the value of the env defined?  It seems for #498 we want just the last indicator to say rooms while the others say structures.

Reply:
oh hmm, i thought one of the countries measured sprayable units as rooms, let me get clarification in the ticket

Reply:
thanks for the explanation @moshthepitt changes made